### ISSUE TITLE
Set a timeout for dropping in-memory `rclone mount` buffer

### ISSUE THREAD

**Issue by agneevX** [rclone/rclone#5592]
#### The associated forum post URL from `https://forum.rclone.org`

https://forum.rclone.org/t/rclone-mount-buffer-not-filling-when-playing-large-files/26405/

#### What is your current rclone version (output from `rclone version`)?

```sh
rclone v1.56.0
- os/version: ubuntu 20.04 (64 bit)
- os/kernel: 5.4.0-1042-raspi (aarch64)
- os/type: linux
- os/arch: arm64
- go/version: go1.16.5
- go/linking: static
- go/tags: none
```

#### What problem are you are trying to solve?

Buffering during playback of high bit rate files on Plex that occur due to the Plex client "dropping" the file, which causes rclone to also drop the in-memory cache for the file.

Since it takes some time to again download the chunks, there's wastage of data due to the dropped buffer and there's buffering visible in the Plex client.

#### How do you think rclone should be changed to solve that?

An option to set a timeout for dropping in-memory buffers set by `--buffer-size` rather than dropping it immediately, when the file is "closed" by the Plex client.

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ivandeex** [comment]
Please point to the bug report for "Plex client dropping the file".
Dropping files is not good.

**Comment by agneevX** [comment]
@ivandeex I've updated the post with a link to the log file.

**Comment by ivandeex** [comment]
Based on
https://forum.rclone.org/t/rclone-mount-buffer-not-filling-when-playing-large-files/26405/17
the enhancement will be useful at least in the **niche case** when:
- rclone device has no usable disks or sdd
- memory is limited
- the dumb stock Plex player or alike is used for streaming

A future patch **must** clearly specify these conditions in the documentation for new flag!

Volunteers welcome!

**Comment by ncw** [comment]
Did you try using `--vfs-cache-mode full`? That will effectively save the chunks to disk so that when the file is re-opened the chunks should be read off disk.

You can then turn `--buffer-size` down to small again.

If you want you can point the vfs cache at a RAM disk and limit its size.

That would be my recommended solution.

**Comment by agneevX** [comment]
> If you want you can point the vfs cache at a RAM disk and limit its size.

Yes, I've tried that but when a single file fills the RAM cache, no other file can be opened, until the current open file is purged. 

[I've written about in in the forum](https://forum.rclone.org/t/rclone-mount-buffer-not-filling-when-playing-large-files/26405/8).

If you've got a better idea, I'd love to hear it.

**Comment by ncw** [comment]
Try it with the VFS cache pointed to disk. Your OS will do caching of files in RAM so it should be just about as quick as the RAM disk for recently used files and has the advantage that it can be bigger.

**Comment by agneevX** [comment]
Right, but there's still activity on the disk and its not entirely done in RAM?

My feature request is to simply not drop the cache when a Plex client drops it and set a timeout for that, similar to the mergerFS `dropcacheonclose` parameter.
